openapi.yaml

```diff
@@ -0,0 +1,787 @@
+          description: The securable to display grants for
+          required: true
+          schema:
+            type: string
+        - name: "subject"
+          in: "query"
+          description: Limit the assignments to the specified subject, optional
+          schema:
+            type: string
+        - $ref: "#/components/parameters/pagingOffset"
+        - $ref: "#/components/parameters/pagingLimit"
+      responses:
+        "200":
+          description: "Display Grants for Securable"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/GrantsResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "500":
+          $ref: "#/components/responses/ServerError"
+  /check-permissions:
+    parameters:
+      - name: "securableIds"
+        in: query
+        description: Id of the securable
+        required: true
+        schema:
+          type: array
+          items:
+            type: string
+          minItems: 1
+      - name: "permissions"
+        in: query
+        required: true
+        description: Permissions that need to be checked. All of these must be present for a successful request.
+        schema:
+          type: array
+          items:
+            type: string
+          minItems: 1
+    get:
+      tags:
+        - 'checks'
+      operationId: checkOwnPermissions
+      responses:
+        "200":
+          description: "Permissions present"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/CheckPermissionsResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "500":
+          $ref: "#/components/responses/ServerError"
+    post:
+      tags:
+        - 'checks'
+      operationId: checkPermissions
+      requestBody:
+        required: true
+        content:
+          'application/json':
+            schema:
+              $ref: "#/components/schemas/AuthorizationPrincipal"
+      responses:
+        "200":
+          description: "Permissions present"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/CheckPermissionsResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "500":
+          $ref: "#/components/responses/ServerError"
+  /users:
+    get:
+      tags:
+        - "subjects"
+      operationId: loadUsers
+      parameters:
+        - $ref: "#/components/parameters/pagingOffset"
+        - $ref: "#/components/parameters/pagingLimit"
+      responses:
+        "200":
+          description: "Get All Users"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/GetUsersResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "500":
+          $ref: "#/components/responses/ServerError"
+  /groups:
```

Comment:
Is this going away since we're using orgs?  Are we changing this to an endpoint for orgs?